src/main/java/io/pravega/connectors/flink/FlinkPravegaWriter.java

```diff
@@ -187,11 +187,11 @@ public void close() throws Exception {
     /**
      * Gauge for getting the scoped name information.
```

Comment:
Since you have changed from "scoped name" to "stream name" (which I think is better), shouldn't you also change in this comment block and in the variable definition inside the class for clarity?

Reply:
I have updated the comment and the variable name